src/main/java/core/basesyntax/serviceimpl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReaderServiceImpl implements ReaderService<List<String>> {
+    private final List<String> lines = new ArrayList<>();
+
+    @Override
+    public List<String> readFromFile(String filePath) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(filePath))) {
+            reader.readLine();
```

Comment:
Skipping the first line without any condition can be problematic. If it's a header line, consider checking for it explicitly or provide a way to configure whether to skip it or not.